traffic_ops/app/db/migrations/20191016_add_dns_challenges.sql

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This file probably needs to be renamed to have a prefix with a later date since another migration has been merged with a later date prefix than this one.

Reply:
done